src/build.py

```diff
@@ -1071,10 +1071,18 @@ def WriteEmscriptenConfig(infile, outfile):
       del os.environ['EM_CONFIG']
 
     # Copy the main system libraries so users don't need to themselves.
-    os.makedirs(GetInstallDir('lib', cache_subdir))
-    for cache_lib in cache_libs:
-      shutil.copy2(os.path.join(EMSCRIPTEN_CACHE_DIR, cache_subdir, cache_lib),
-                   GetInstallDir('lib', cache_subdir, cache_lib))
+    packaging_dir = GetInstallDir('lib', cache_subdir)
+    if not os.path.exists(packaging_dir):
```

Comment:
could use os.path.isdir instead of os.path.exists

Reply:
Actually is os.makedirs a no-op for pre-existing directories?

Reply:
It's not a no-op - it throws for me locally during testing, but not on the bots where I guess they start fresh.

Reply:
ah ok, then isdir is probably the right thing.